ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -326,6 +326,7 @@
   timelineReplace: [
     {
       'locale': 'de',
+      'missingTranslations': true,
```

Comment:
What changed such that German has something missing?

Reply:
I understand that this line `'missingTranslations': true,` should be there, when there are lines to translate. Am I mistaken?
![e5s de](https://user-images.githubusercontent.com/61565834/84476612-af9f2f00-ac8e-11ea-9452-e0f6805de58e.PNG)

Reply:
but is there a missing translation in the translation section? 
only than (afaik) we need the missing translation flag

Reply:
afaik ? i not understand. My apologies.

Reply:
afaik : As far as I know

Reply:
This region just shows the missing translation of alert texts.
In my opinion, you don't need a `missingTranslation` in `timelineReplace`, because there is no "missing translations" in `timelineReplace` region, just remove it as Akurosia said.

Reply:
Wouldn't it be interesting to point out that there are untranslated lines even if it doesn't concern `replaceText` or `replaceSync`?

Reply:
The point of the "missing translation" flag was to try to make it more obvious from reading the code whether or not a language is complete for a given fight.  It's easy to look at a single alertText or regex and see that a language is missing.  It's hard to look at a timelineReplace block and see that something is missing.

Reply:
must i remove all lines `missingtranslation` added ?

Reply:
`npm run test` (which runs automatically when you upload a pull request) will tell you when it it necessary if it is missing.  Please don't add it to anything unless that test has told you to.